AU2/plugins/CorePlugin.py

```diff
@@ -381,6 +406,27 @@ def answer_core_plugin_update_assassin(self, html_response_args: Dict):
             return_components += p.on_assassin_update(assassin, html_response_args)
         return return_components
 
+    def gather_dead_non_police(self) -> List[str]:
+        death_manager = DeathManager(perma_death=True)

```

Comment:
I see why you've done it this way, but `perma_death=True` is incorrect behavior for May Week games. This is fine in some plugins (e.g., Competency and Targeting) that explicitly are not used for May Week. How about setting `perma_death=True` iff the TargetingPlugins is enabled?

Reply:
Ah that's a good point.

It feels kind of hacky for `CorePlugin` to be checking for a specific plugin, but a better solution doesn't come to mind...

Reply:
Maybe since `perma_death=False` just makes the list empty (so effectively disables the export) the export itself should be part of `TargetingPlugin`?